devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy

```diff
@@ -0,0 +1,501 @@
+/**
+ *	Copyright 2020 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+import java.lang.Math
+
+metadata {
+	definition (name: "Qubino Flush Shutter", namespace: "qubino", author: "SmartThings", ocfDeviceType: "oic.d.blind", mcdSync: true) {
+		capability "Window Shade"
+		capability "Window Shade Level"
+		capability "Power Meter"
+		capability "Energy Meter"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Configuration"
+
+		capability "Switch Level"
```

Comment:
Since this is a new device, I think we should not add `Switch Level` to the capabilities. It will just make more work to remove it later.

Reply:
By the way, here will be an update to allow for Window Shade Level in UI metadata (instead of needing Switch Level) this week if there hasn't been one already.